source/Playnite/Localization/LocSource.xaml

```diff
@@ -542,6 +542,9 @@ If disabled, games that use any item in any filter will be included in the view.
     <sys:String x:Key="LOCGameIsUnInstalledTitle">Not installed</sys:String>
     <sys:String x:Key="LOCGameHiddenTitle">Hidden</sys:String>
     <sys:String x:Key="LOCGameFavoriteTitle">Favorite</sys:String>
+    <sys:String x:Key="LOCGameHdrTitle">HDR</sys:String>

```

Comment:
Should be described as "Enable HDR support" or something similar.

Reply:
Done